chat-plugins/room-events.js

```diff
@@ -94,11 +94,53 @@ exports.commands = {
 		help: function (target, room, user) {
 			return this.parse('/help roomevents');
 		},
+		sortby: function (target, room, user) {
+			if (!room.chatRoomData) return this.errorReply("This command is unavailable in temporary rooms.");
+			if (!room.events || !Object.keys(room.events).length) {
+				return this.errorReply("There are currently no planned upcoming events for this room.");
+			}
+			if (!this.can('ban', null, room)) return false;
+			let multiplier = 1;
+			let columnName = "";
+			let order = "";
+			if (target.split(target.includes('|') ? '|' : ',') === target) {
```

Comment:
this is always going to be false, since you're comparing an array to a string, i think you just want something like ``if (target.split(etc).length === 1)``